### DISCUSSION TITLE
PSX Memory Card Reader

### DISCUSSION THREAD

**Discussion by Androxilogin** [sanni/cartreader#259] [Ideas & Suggestions]
I've brought this up in the past, **[see here](https://forum.arduino.cc/t/rom-reader-for-super-nintendo-super-famicom-game-cartridges/155260/1031)** but since then I've had a lot of different projects come my way. I also upgraded my PC that I had the PSX.ino files and everything listed in the previous post that Sanni uploaded to get me started. Unfortunately the Dropbox link has also expired.

Recently I've been looking into trying to get this going again so yesterday I put together a prototype to add it to the cart reader.

**[Here is what I ended up with so far.](https://i.imgur.com/qOMDDry.jpeg)**

**[Here](https://i.imgur.com/OqgKbie.jpeg)** shows the alignment of where the pins will line up with the board.

Pins seem to line up great and it has a satisfying "locked in" feel to it without gripping the card too aggressively.

**[And this](https://i.imgur.com/IuL250E.jpeg)** is an MemCarduino attached to an Arduino Nano as the original developer had in mind. I plan to make a case for this as well as soon as I figure out what kind of "hooking" pins to use. Anyone have any suggestions?

I've purchased **[these](https://www.ebay.com/itm/153301627219)** in the past for other projects but they are entirely too flexible and the tabs should be 1.75 at width. 

Anyone have any suggestions or would like to help out with this addon?

**EDITED** to include images more proper in markdown.

**Comment by sanni** [comment]
Looks great. 👍 
I have searched my PC but could not find the PSX addon I started porting some time ago either.

**Reply by Androxilogin** [reply]
**UPDATE:** [**I found it**](https://mega.nz/file/idg2TKYD#5SCUiD1Li48DFLS7u3HIWbxW19fNYs-2DrZ9VCo84lM) while searching for large files to see what I could get rid of in a folder called "Sort Them Out".

**Comment by Androxilogin** [comment]
I can't believe I don't have a copy of it somewhere. Sadly I don't understand the coding enough to know much about getting this to work. Hopefully someone else here will get interested in this. Thanks for initially porting it.
